src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -110,6 +110,14 @@ export const MultiHopTrade = memo(
     }, [dispatch, routeBuyAsset, routeSellAsset, sellAmountCryptoBaseUnit, isInitialized])
 
     useEffect(() => {
+      // Poor man's check for input route. Do not rewrite URL on those routes, or problems.
+      const isInputRoute =
+        !location.pathname.includes(TradeRoutePaths.QuoteList) &&
+        !location.pathname.includes(TradeRoutePaths.Quotes) &&
```

Comment:
q: What is this route? It doesnt seem consumed anywhere

Reply:
https://github.com/shapeshift/web/pull/9063